src/Packetery/Module/Plugin.php

```diff
@@ -1144,8 +1155,13 @@ public function addPluginRowMeta( array $links, string $pluginFileName ): array
 	 *
 	 * @return array
 	 */
-	public function add_shipping_method( array $methods ): array {
-		$methods[ ShippingMethod::PACKETERY_METHOD_ID ] = ShippingMethod::class;
+	public function addShippingMethods( array $methods ): array {
+		if ( $this->optionsProvider->isWcCarrierConfigEnabled() ) {
+			$methods = $this->shippingProvider->addMethods( $methods );
```

Comment:
nepřepisoval bych takhle jednu proměnou

buď 
$methods = $this->shippingProvider->addMethods($this->shippingProvider->sortMethods( $methods ));
ale to zase není taky idealní pro debug.

anebo idealně to vrátit to jiné proměnné třeba $unsortedMethods a tu pak poslat do sortMethods

takhle jsem se nad tím musel pozastavit, protože naprvní pohled jsem myslel, že to jsou duplicitní řádky